src/test/index.ts

```diff
@@ -148,5 +148,11 @@ export const test = _baseTest.extend<PlaywrightTestArgs & PlaywrightTestOptions,
   page: async ({ context }, use) => {
     await use(await context.newPage());
   },
+
+  baseURL: [async ({ }, use) => {
+    if (!process.env.PW_BASE_URL)
+      throw new Error('A web server must be specified to use baseURL. See `webServer` in the config options, or set the PW_BASE_URL environment variable.');
```

Comment:
Note that you can just do `use: { baseURL: 'foo' }` in the config, so perhaps we do not need the environment variable?